pkg/resource/interfaces.go

```diff
@@ -82,6 +82,13 @@ type Reclaimer interface {
 	GetReclaimPolicy() v1alpha1.ReclaimPolicy
 }
 
+// A CredentialsSecretReferencer may refer to a credential secret to an arbitrary
+// namespace.
+type CredentialsSecretReferencer interface {
```

Comment:
Since these methods are not currently present on existing `Provider` types, do you plan to implement them in each stack? We should probably generate them using `crossplane-tools/angryjet`

Reply:
Yes, after getting a thumbs-up from you guys I plan to implement them in `crossplane-tools/angryjet`

Reply:
Nice!